style/javascript/README.md

```diff
@@ -0,0 +1,18 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
```

Comment:
No, I'd say it means do things like `class Router extends Backbone.Router` instead of `var Router = Backbone.Router.extend`.

I don't think we should be using Backbone anymore, but I chose it since it should be familiar to a lot of us.